php/libraries/NDB_Caller.class.inc

```diff
@@ -476,7 +476,9 @@ class NDB_Caller
         }
 
         // save instrument form data
-        $success = $instrument->save();
+        if (empty($_POST['login'])) {
```

Comment:
Is this the right place for this? Wouldn't it make more sense to put it up in the load method so it applies to all modules? (In fact, it looks like there's a missing "return" up around line 93, after it sets the Location header.. it probably never should have gotten to this line in the first place.)